test/fixtures/e2e/amp-viewer-integration/viewer.html

```diff
@@ -7,7 +7,7 @@
       name="viewport"
       content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui"
     />
-    <script src="/dist/v0/examples/amp-viewer-host.max.js"></script>
+    <script src="https://cdn.ampproject.org/v0/examples/amp-viewer-host.js"></script>
```

Comment:
Since this isn't an integration test fixture (double check me on this) - thoughts on leaving this unchanged?

Reply:
yep you're right. ill revert